controller/src/main/java/org/jboss/as/controller/AttributeMarshallers.java

```diff
@@ -55,12 +55,18 @@ public boolean isMarshallable(AttributeDefinition attribute, ModelNode resourceM
 
         @Override
         public void marshallAsElement(AttributeDefinition attribute, ModelNode resourceModel, boolean marshallDefault, XMLStreamWriter writer) throws XMLStreamException {
-            if (!resourceModel.hasDefined(attribute.getName())) {
+            if (!resourceModel.has(attribute.getName())) {
```

Comment:
Another reason why this method (and similarly the List marshaller one) look bit funky is becouse of bugs in various jaxb parsers.

the whole
```
if (!resourceModel.isDefined()){//this is scenario to handle WFCORE-1448 cases
   writer.writeEmptyElement(wrapper);
   return;
}
```
is only needed as some parsers complain if you do 
```
writer.writeStartElement(wrapper);
writer.writeEndElement();
```
even if semantically the do exactly the same thing.